src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Email/SmtpConfigurationType.php

```diff
@@ -65,6 +65,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'empty_data' => '',
                 'label' => $this->trans('SMTP password', 'Admin.Advparameters.Feature'),
                 'help' => $this->trans('Leave blank if not applicable.', 'Admin.Advparameters.Help'),
+                'attr' => [
+                    'autocomplete' => 'new-password',
```

Comment:
Maybe we should simply put it to `off` instead of `new-password`? Do we really want Chrome to store the SMTP password?

Reply:
It's a bit of a hack, because some browsers ignore the off property.

Reply:
Then maybe add a little comment in the code? I will not be the only one to ask 😄

Reply:
"off" isn't working in Chrome. :/ So I used "new-password" ...

I added comment there :)
Good now?